katello/service-wait

```diff
@@ -57,7 +57,7 @@ before_start() {
 # after start or restart
 after_start() {
   case "$SERVICE" in
-    tomcat6|tomcat7)
+    tomcat|tomcat6|tomcat7)
```

Comment:
I think tomcat7 can actually be removed, not sure that's the name of any service